src/rpc/client/ts/package.json

```diff
@@ -13,31 +13,33 @@
   "author": "Shane Dell, Davin Shearer, John Wass",
   "license": "Apache-2.0",
   "scripts": {
+    "build": "webpack --mode=production",
     "compile-src": "run-script-os",
-    "compile-src:default": "./compile-proto.sh",
-    "compile-src:windows": "./compile-proto.bat",
-    "precompile": "node -e \"require('./scripts/package.ts').genClientVersion('./src/client_version.ts')\"",
-    "compile": "tsc -p ./",
+    "compile-src:default": "./compile-proto.sh && ./compile-proto.sh --out",
+    "compile-src:windows": "./compile-proto.bat && ./compile-proto.bat --out",
```

Comment:
What is the benefit of calling this script twice?  Why not do the compile in a single call instead of in 2 steps?